lib/mix/tasks/nerves.system.shell.ex

```diff
@@ -70,11 +65,21 @@ defmodule Mix.Tasks.Nerves.System.Shell do
 
     {build_runner, _opts} = pkg.build_runner
 
-    build_runner.system_shell(pkg)
+    {cmd, initial_input} = build_runner.system_shell(pkg)
+
+    if String.to_integer(System.otp_release()) < 26 do
+      # Legacy shell support
+      Mix.Nerves.Shell.open(cmd, initial_input)
+    else
+      Mix.Nerves.IO.shell_warn("shell start deprecated", """
+      OTP 26 made several changes to the serial interface handling. Unfortunately, this
+      a regression in preventing the Nerves tooling from starting a system sub-shell.
 
-    # Set :user back to the real one
-    Process.register(user, :user)
+      However, the build directory and all needed pieces have been created and can be
+      accessed by manually running the command below:
 
-    :ok
+      #{[IO.ANSI.reset(), cmd]}
```

Comment:
Is this how the workflow works?

```suggestion
      #{[IO.ANSI.reset(), cmd]}
      
      When you're in the container, run `make` to build the system as is or run `make menuconfig`
      to modify it. When done, exit the container and run `mix firmware` again.
```